templates/cluster.yml.j2

```diff
@@ -15,6 +15,13 @@ spec:
   docker:
     insecureRegistry: 100.64.0.0/10
     logDriver: ""
+  kubeAPIServer:
```

Comment:
Keep in mind that this is a publicly available Github project that tries to be generic - not only for us.
E.g.:

**`defaults/main.yml`**
```yml
kops_default_kube_api_server: {}
```

**`templates/cluster.yml.j2`**
```yml
    kubeAPIServer: "{{ kops_default_kube_api_server }}"
```

Those values will then be set in our playbook repositories.

Additionally this variable must be documented in the README.md